src/Bundle/Resources/config/services/integrations/doctrine/orm.xml

```diff
@@ -20,6 +20,24 @@
     <services>
         <defaults public="true" />
 
+        <service id="sylius.doctrine_orm.metadata.resource.metadata_collection_factory"
```

Comment:
```suggestion
        <service id="sylius.resource_metadata_collection.factory.doctrine.orm"
```
Would make it easier to find since it'd be in line with the decorated service